exonum/src/storage/proof_map_index/key.rs

```diff
@@ -91,25 +91,28 @@ impl DBKey {
         }
     }
 
+    #[doc(hidden)]
     pub fn from(&self) -> u16 {
         self.from
     }
 
     // TODO: terrible hack, try to remove this
+    #[doc(hidden)]
     pub fn set_from(&mut self, from: u16) {
         self.from = from
     }
 
+    #[doc(hidden)]
     pub fn to(&self) -> u16 {
         self.to
     }
 
-    /// Length of the `DBKey`
+    /// Returns length of the `DBKey`.
```

Comment:
This comment explains nothing, because the _length_ of the DBKey is 34 bytes :-)

It seems to return the number of significant **bits** in the 'data' field. Must be from 0 for root node of the MPT and up to 256 for a leaf node. 

On top of that value of 'to' in DBKey is not the same as the value of the last byte of the key in the underlying storage (which is from 0 to 255 for branch nodes, and always zero for leaves). See `impl StorageKey for DBKey` for more.